### PR TITLE
UI redesign

### PR DESCRIPTION
**Pull request by karthik-durai**
# Pull Request

## Changes

1. Implement new UI design

## Checklist

- [X] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [X] The changes have been tested locally.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -225,7 +225,7 @@ onMounted(() => {
   <div class="flex flex-col h-full">
     <div
       v-show="expandWallet || app.expandRestoreScreen"
-      class="flex flex-col h-full bg-white-300 dark:bg-black-300 overflow-hidden"
+      class="flex flex-col h-full bg-[#EFEFEF] dark:bg-black-300 overflow-hidden"
```
Comment (shrinathprabhu) [inline src/App.vue:228]: Please add this to `tailwind-config`

File: src/assets/images/light/AppFooter.vue [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  >
+    <nav v-if="userStore.isLoggedIn" class="flex flex-grow justify-between">
+      <router-link
+        :to="{ name: 'home' }"
+        title="Assets"
+        class="flex flex-col justify-center items-center gap-1"
+      >
+        <div
+          class="w-xxxl h-xxl rounded-md flex items-center justify-center"
+          :class="{ 'bg-[#1D2A31] dark:bg-[#F7F7F7]': isCurrentRoute('home') }"
```
Comment (shrinathprabhu) [inline src/assets/images/light/AppFooter.vue:45]: Maintain these colors in tailwind config

File: src/assets/images/light/AppFooter.vue [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+            :src="getIcon('tokens-icon', 'home')"
+            alt="home"
+            class="w-xl h-xl"
+          />
+        </div>
+        <span
+          class="text-xs font-norma"
+          :class="
+            isCurrentRoute('home')
+              ? 'text-[#1D2A31] dark:text-[#F7F7F7]'
```
Comment (shrinathprabhu) [inline src/assets/images/light/AppFooter.vue:57]: Colors in tailwind config

File: src/assets/images/light/AppFooter.vue [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+          :class="{ 'bg-[#1D2A31] dark:bg-[#F7F7F7]': isCurrentRoute('home') }"
+        >
+          <img
+            :src="getIcon('tokens-icon', 'home')"
+            alt="home"
+            class="w-xl h-xl"
+          />
+        </div>
+        <span
+          class="text-xs font-norma"
```
Comment (shrinathprabhu) [inline src/assets/images/light/AppFooter.vue:54]: what is font-norma?

File: src/assets/images/light/AppFooter.vue [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+        >
+      </router-link>
+      <router-link
+        :to="{ name: 'Nfts' }"
+        title="NFTs"
+        class="flex flex-col justify-center items-center gap-1"
+      >
+        <div
+          class="w-xxxl h-xxl rounded-md flex items-center justify-center"
+          :class="{ 'bg-[#1D2A31] dark:bg-[#F7F7F7]': isCurrentRoute('Nfts') }"
```
Comment (shrinathprabhu) [inline src/assets/images/light/AppFooter.vue:70]: Maintain colors in tailwind config

File: src/components/SendTransaction.vue [hunk 6]
```diff
@@ -213,6 +224,24 @@ function calculateCurrencyValue(value) {
   <div v-if="loader.show" class="flex justify-center items-center flex-1 p-4">
     <AppLoader :message="loader.message" />
   </div>
+  <div v-else-if="shrinkMode" class="flex justify-between p-3">
```
Comment (shrinathprabhu) [inline src/components/SendTransaction.vue:227]: What is shrinkMode?

Comment (karthik-durai) [reply src/components/SendTransaction.vue:227]: shrink mode is to show pending tasks in the activities tab

File: src/assets/images/light/AppFooter.vue (hunk 4 above)
Comment (karthik-durai) [reply src/assets/images/light/AppFooter.vue]: [fixed](https://github.com/arcana-network/wallet-ui/pull/693/commits/f997475ff9d23b36ea41e8217957c2029e930b46)

File: src/App.vue (hunk 1 above)
Comment (karthik-durai) [reply src/App.vue:228]: [added](https://github.com/arcana-network/wallet-ui/pull/693/commits/6b3ff70f5914dc34065be1596814c2017a9c5507)

File: src/assets/images/light/AppFooter.vue (hunk 2 above)
Comment (karthik-durai) [reply src/assets/images/light/AppFooter.vue]: [added](https://github.com/arcana-network/wallet-ui/pull/693/commits/6b3ff70f5914dc34065be1596814c2017a9c5507)

File: src/assets/images/light/AppFooter.vue (hunk 3 above)
Comment (karthik-durai) [reply src/assets/images/light/AppFooter.vue]: [added](https://github.com/arcana-network/wallet-ui/pull/693/commits/6b3ff70f5914dc34065be1596814c2017a9c5507)

File: src/assets/images/light/AppFooter.vue (hunk 5 above)
Comment (karthik-durai) [reply src/assets/images/light/AppFooter.vue]: [added](https://github.com/arcana-network/wallet-ui/pull/693/commits/6b3ff70f5914dc34065be1596814c2017a9c5507)

**Review by shrinathprabhu — APPROVED** [review]

**Pull request merged**